features/cellular/framework/targets/UBLOX/AT/UBLOX_AT_CellularStack.cpp

```diff
@@ -201,7 +201,8 @@ nsapi_size_or_error_t UBLOX_AT_CellularStack::socket_sendto_impl(CellularSocket
             }
             _at.cmd_start_stop("+USOWR", "=", "%d%d", socket->id, blk);
 
-            (void)poll(&fhs, 1, 50);
+            rtos::ThisThread::sleep_for(50);
+            _at.read_bytes(&ch, 1);
```

Comment:
After the @ prompt reception, wait for a minimum of 50 ms before sending data is not happening.

Reply:
added wait before reading @.